index.html

```diff
@@ -0,0 +1,122 @@
+            <a class="nav-item-desk" href="">News</a>
+            <a class="nav-item-desk" href="">CC Campaign</a>
+        </nav>
+        <div><h2 class="logo hide-from-mobile">Logo</h2></div>
+    </header>
+
+    <section class="hero">
+            <img class="hero-bg" src="images/hero-bg.jpg" alt="">
+            <h2 class="welcome-title">"Hello! Aspiring people"</h1>
+            <h1 class="title">COMPUTER SCIENCE<br>FOR ALL<br>
+            FREE TICKETS 2022</h1>
+            <article class="intro">
+                <p class="intro-details">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis veritatis corporis iusto, quasi deleniti quisquam voluptates non dolorem magni fugit ducimus consectetur dolore. Laboriosam iste exercitationem at hic placeat! Est.</p>
+            </article>
+            <p class="event-time">27.11.2022(Sat)-(Sun)</p>
+            <p class="event-place">@ Online Zoom Hosted session</p>
+       
+    </section>
+
+    <section class="main-program">
+         <img class="main-program-bg" src="images/main-bg.jpg" alt="">
+        <h2 class="programs-title">Main Program</h2>
+        <hr class="separator">
+        <div class="programs-container mobile-container">
+            <div class="programs">
+                <img class="programs-icon" src="images/binary.png" alt="">
+                <h3 class="programs-titles">Binary</h3>
+                <p class="programs-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio, error!</p>
+            </div>
+            <div class="programs">
+                <img class="programs-icon" src="images/algorithm.png" alt="">
+                <h3 class="programs-titles">Algorithms</h3>
+                <p class="programs-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio, error!</p>
+            </div>
+            <div class="programs">
+                <img class="programs-icon" src="images/data-structures.png" alt="">
+                <h3 class="programs-titles">Data Structures</h3>
+                <p class="programs-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio, error!</p>
+            </div>
+            <div class="programs">
+                <img class="programs-icon" src="images/memory.png" alt="">
+                <h3 class="programs-titles">Memory</h3>
+                <p class="programs-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio, error!</p>
+            </div>
+            <div class="programs">
+                <img class="programs-icon" src="images/computer.png" alt="">
+                <h3 class="programs-titles">Computer Architecture</h3>
+                <p class="programs-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio, error!</p>
+            </div>
+        </div>
+        <button class="join-btn">Join CS for All Course 2022</button>
+        <a class="desktop-btn" href="">SEE THE WHOLE PROGRAM</a>
```

Comment:
[OPTIONAL] Nice work on the project so far. :+1:  However, remember that to be a very good programmer you need to pay attention to detail and be open-minded for advice and suggestions. This project allows for small adjustments like text or images, but you should ensure that you follow the design guidelines in terms of colors, typography (like font face, size, and font-weight) as well as the layout composition and space between elements. You can consider going through your project and making these adjustments in order to make your project look more like the design guidelines provided. :pray:

Reply:
Thank you Jerry I really appreciate it, have a nice evening